static/admin/config.yml

```diff
@@ -26,10 +26,13 @@ collections:
       - {label: Image, name: image, widget: image}
       - {label: Subheading, name: subheading, widget: string}
       - {label: About, name: about, widget: string}
-      - {label: Button, name: button, widget: object, fields: [
-          {label: "Button Label", name: buttonText, widget: string},
-          {label: "Button Link", name: buttonLink, widget: string}
-        ]}
+      - label: "Concert Attire"
+        name: concertAttire
+        widget: object
+        fields:
+          - {label: "Concert Attire Heading", name: attireHeading, widget: string, default: 'Concert Attire'}
+          - {label: "Concert Attire Description", name: attireContent, widget: text, default: 'Brief description...'}
+          - {label: "Concert Attire Content", name: attireDrawer, widget: text, default: 'More text here...'}
```

Comment:
@GusTheBusNG Does this fix the issue?

Reply:
Yeah I think so, I'll more into it on the `bullet-proof` branch